src/main/kotlin/com/autonomousapps/extension/Issue.kt

```diff
@@ -88,7 +90,18 @@ open class Issue @Inject constructor(
    * tells the plugin to exclude those dependencies in the final advice.
    */
   fun exclude(vararg ignore: String) {
-    excludes.addAll(ignore.toSet())
+    excludes.addAll(ignore.map { Exclusion.ExactMatch(it) }.toSet())
+  }
+
+  /**
+   * All elements matching the provided patternw ill be filtered out of the final advice. For example:
+   * ```
+   * excludeRegex(".*:internal$")
+   * ```
+   * tells the plugin to exclude any subproject named ":internal" in the final advice.
+   */
+  fun excludeRegex(@Language("RegExp") vararg patterns: String) {
+    excludes.addAll(patterns.map { pattern -> Exclusion.PatternMatch(Regex(pattern, MULTILINE)) })
```

Comment:
does MULTILINE really make sense in this context?

Reply:
Nope, you're right, it works without it.  😆